src/libYARP_math/CMakeLists.txt

```diff
@@ -24,8 +24,6 @@ if (YARP_HAS_MATH_LIB)
                      include/yarp/math/Vec2D.h
                      include/yarp/math/FrameTransform.h)
 
-  set(YARP_math_IMPL_HDRS include/yarp/math/impl/mt.h)
-
```

Comment:
Please leave this variable here, but empty

```diff
-  set(YARP_math_IMPL_HDRS include/yarp/math/impl/mt.h)
+ set(YARP_math_IMPL_HDRS )
```

This is for coherence with the other files

Reply:
:+1: CMakeLists restored in https://github.com/robotology/yarp/pull/1444/commits/8d494a30f61b8fd2b613b7eff473cbb067338166